### PR TITLE
improvements: created User profile model and serializer

### PR DESCRIPTION
**Pull request by nextwebb**
Worked on Authenticated user settings model and serializer  #4

### CODE REVIEW

**Review by omidiora — DISMISSED** [review]

**Comment by simplytunde** [comment]
You cant  merge the PR until all the test pass. Also, you may have to install Pillow for the CI test.

**Review by omidiora — COMMENTED** [review]
In your models.py. i  think  it should be a custom user models

**Comment by nextwebb** [comment]
@omidiora not using default user model because of "username property" shouldnt be a deal breaker

**Review by omidiora — APPROVED** [review]

File: api/models.py [hunk 1]
```diff
@@ -1,3 +1,16 @@
 from django.db import models
+from django.contrib.auth.models import User
+# from django.core.validators import MaxLengthValidator, MinLengthValidator
+
+class Profile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE, null=False, blank=False, unique=True,
+                                related_name = 'user')
+    facebook_user = models.CharField(max_length=50, blank=True, null=True, unique=True )
```
Comment (simplytunde) [inline api/models.py:8]: I will like to understand the reasoning behind adding `facebook_user`.

File: api/urls.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+from rest_framework import routers
+from django.conf.urls import include
+from .views import UserViewSet, ProfileViewSet
+
+router = routers.DefaultRouter()
+router.register('user', UserViewSet)
+router.register('profile', ProfileViewSet)
+
+urlpatterns = [
+    path('v1/', include(router.urls)),
```
Comment (simplytunde) [inline api/urls.py:11]: Think of how to parameterize the API version.

**Comment by simplytunde** [comment]
Once some of my comments are reviewed, I will go ahead with approval.

**Comment by nextwebb** [comment]
> You cant merge the PR until all the test pass. Also, you may have to install Pillow for the CI test.

i have installed pillow package

**Review by omidiora — COMMENTED** [review]
the reason why i approve the pull request    is that  it match the django documentation
    https://www.django-rest-framework.org/api-guide/viewsets/

**Review by simplytunde — APPROVED** [review]

File: api/models.py (hunk 1 above)
Comment (nextwebb) [reply api/models.py:8]: Facebook social Auths

**Pull request merged**